packages/fx-core/src/common/samples-config.json

```diff
@@ -207,6 +207,16 @@
             "configuration": "Ready for debug",
             "suggested": false
         },
+        {
+            "id": "teams-videoapp-sample",
+            "title": "Video Filter App Sample",
```

Comment:
It is duplicate word "sample" in "sample" gallery. Is "minimal Teams video app" better?

Reply:
Updated.